City-of-Helsinki/drupal-helfi-tyollisyyspalvelut-manuaali
# TP-1433 Fix some printing styles

**Actions necessary for applying the changes:**
drush deploy

**Testing instructions:**

Go to a service page, print it (both the client view and the work instructions) and check the following:
- [x] Service languages and their levels are now printed inline.
- [x] Language levels are not links anymore (as there is no point trying to open modals from printed page).
- [x] Lists are now more readable: unordered items are printed using the normal bullet points (as background colors are often omitted from prints) and ordered are printed showing a border around the number.
- [x] There is only one icon for file items (the download icon is hidden).
- [x] Accordions inside rich text fields are open and the open-close buttons are hidden.
- [x] The "time and place" cards have nice margins.
- [x] Try to make sure there is no weird page breaks and problems with page margins (e.g. the content is not "cut off" when printing).

**Review checklist:**
